### PR TITLE
Localize edit command and binary caching

### PR DESCRIPTION
**Pull request by Thomas1664**

### CODE REVIEW

File: src/vcpkg/commands.edit.cpp
Comment (ras0219-msft) [inline src/vcpkg/commands.edit.cpp:89]: I think this will print "error: " twice after https://github.com/microsoft/vcpkg-tool/pull/553. That PR also adds additional validation for localization changes, so let's wait until that one gets merged.

**Comment by ras0219-msft** [comment]
Thanks for the PR! Waiting for https://github.com/microsoft/vcpkg-tool/pull/553 first.

File: src/vcpkg/commands.edit.cpp
Comment (Thomas1664) [reply src/vcpkg/commands.edit.cpp]: You're right

**Review by BillyONeal — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for your help!

**Pull request merged**
